crates/egui/src/ui.rs

```diff
@@ -815,6 +815,10 @@ impl Ui {
         &self.placer
     }
 
+    pub fn placer_mut(&mut self) -> &mut Placer {
```

Comment:
You should probably make `player()` public too then?

Reply:
Making this public and mutable means the user can change layout in the middle of a UI, which is something new, and might break the expectations of other widgets and containers. I suggest we should add a strong "here be dragons" warning here